### PR TITLE
Added new updates

### PR DESCRIPTION
**Pull request by ksr30**

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -412,172 +422,172 @@ pub type Executive = frame_executive::Executive<
...
+				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef780d41e5e16056765bc8461851072c9d7").to_vec().into(),
+			];
+
+			let mut batches = Vec::<BenchmarkBatch>::new();
+			let params = (&config, &whitelist);
+
+			add_benchmark!(params, batches, frame_system, SystemBench::<Runtime>);
+			add_benchmark!(params, batches, pallet_balances, Balances);
+			add_benchmark!(params, batches, pallet_timestamp, Timestamp);
+			add_benchmark!(params, batches, pallet_template, TemplateModule);
```
Comment (Gauthamastro) [inline runtime/src/lib.rs:587]: Don't we need to add back the Polkadex Fungible Assets??? @ksr30

Comment (Gauthamastro) [reply runtime/src/lib.rs:587]: Also, remove Template Module

**Review by Gauthamastro — CHANGES_REQUESTED** [review]

File: runtime/src/lib.rs (hunk 1 above)
Comment (Gauthamastro) [inline runtime/src/lib.rs:587]: Don't we need to add back polkadex_fungible_assets and remove Template

**Review by Gauthamastro — APPROVED** [review]

File: runtime/src/lib.rs (hunk 1 above)
Comment (ksr30) [reply runtime/src/lib.rs:587]: we didn't start benchmarking yet ... ignore for now

**Pull request merged**